modules/ROOT/pages/dashboards/integrations.adoc

```diff
@@ -0,0 +1,4 @@
+= Integrations
```

Comment:
This will probably not happen until later this year, we can leave this out for Q1.